packages/pigeon/platform_tests/android_unit_tests/android/app/src/test/java/com/example/android_unit_tests/PigeonTest.java

```diff
@@ -0,0 +1,85 @@
+package com.example.android_unit_tests;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import io.flutter.plugin.common.BinaryMessenger;
+import io.flutter.plugin.common.MessageCodec;
+import io.flutter.plugin.common.StandardMessageCodec;
+import java.nio.ByteBuffer;
+import java.util.Map;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+public class PigeonTest {
+  private MessageCodec<Object> mCodec = StandardMessageCodec.INSTANCE;
```

Comment:
for consistency, we can skip the hungarian notation https://google.github.io/styleguide/javaguide.html#s5-naming

Reply:
I always preferred google's other java style: https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions

I'll change this if you want.